### ISSUE TITLE
Enable classic MIDI input

### ISSUE THREAD

**Issue by luisfcorreia** [probonopd/MiniDexed#103]
It would be nice to be able to send classic MIDI data through RX pin

It only needs a simple optocoupler circuit for acccepting notes and other data.

**Comment by luisfcorreia** [comment]
maybe already present, will test it soon

(https://github.com/probonopd/MiniDexed/blob/main/src/serialmididevice.cpp)

**Comment by probonopd** [comment]
Yes, it is already present software-wise. You still need to build the optocoupler circuit. Please let us know which exact circuit you are using if you succeed! (I had played with this very briefly but never got it to work electrically.)
